scripts/apps/users/components/TextWithMentions.tsx

```diff
@@ -13,7 +13,7 @@ const mentionRegexp = /@\[([^\]]+)\]\((desk|user):([^\)]+)\)/g;
  * @description Displays a text containing mentions.
  */
 export const TextWithMentions: React.StatelessComponent<any> = ({children, ...props}) => {
-    const msg = children;
+    const msg = props.data ? props.data : children;
```

Comment:
can you add TypeScript types to `props` of this component?

Given `children` can only be a string and that there's only one usage of the component, it'd be better to use a standard prop like `message: string`. We'd then get better enforcement of types by TypeScript and it would work well with angular too without the need to accept the same data via 2 props like now.